marmoset/dhcp/dhcp_config.py

```diff
@@ -17,8 +17,14 @@ def __init__(self, mac, ip_address, gateway=None, networkmask=None):
 
         self.set_settings(True, mac, ip_address, gateway, networkmask)
 
-    def set_settings(self, allow_none_value_for_not_required_parameter=False, mac=None, ip_address=None,
-                     gateway=None, networkmask=None):
+    def set_settings(
+            self,
+            allow_none_value_for_not_required_parameter=False,
```

Comment:
The name is a bit long, yes. But IMO it describes very good the usage of the variable. Not sure if we want to rename that. Any other opinions here?